app/src/main/java/com/duckduckgo/app/bookmarks/ui/FavoritesAdapter.kt

```diff
@@ -60,14 +60,15 @@ class FavoritesAdapter(
 
     fun setItems(
         favoriteItems: List<FavoriteItem>,
+        filteringMode: Boolean = false,
     ) {
         val generatedList = generateNewList(favoriteItems)
         val diffCallback = DiffCallback(old = this.favoriteItems, new = generatedList)
         val diffResult = DiffUtil.calculateDiff(diffCallback)
         this.favoriteItems.clear().also { this.favoriteItems.addAll(generatedList) }
         diffResult.dispatchUpdatesTo(this)
 
-        if (favoriteItems.isEmpty()) {
+        if (filteringMode || favoriteItems.isEmpty()) {
```

Comment:
this whole `if` can be removed, right? it's guarding against nothing; it used to do an _early return_ to prevent some other code from running i'm guessing but is redundant now

Reply:
You're right, that doesn't need to be there anymore, I'll remove 👍